### PR TITLE
 Updated enchant to do a better job of cleaning the brazier

### PR DESCRIPTION
**Pull request by dr-leustyin**

### CODE REVIEW

File: enchant.lic [hunk 1]
```diff
@@ -117,17 +124,28 @@ class Enchant
...
+    when 'You prepare to clean off the brazier'
+      bput("clean #{@brazier}","a massive ball of flame jets forward and singes everything nearby")
+    end
+  end
+  
+  def empty_brazier
+    case bput("look on #{@brazier}",/On the brazier you see (.*)./,'There is nothing')
+    when /On the brazier you see (.*)./
+      items =  Regexp.last_match(1)
+      items = items.split(" and ")
```
Comment (rpherbig) [inline enchant.lic]: Does `DRC.list_to_nouns(items)` do the same thing?

File: enchant.lic [hunk 2]
```diff
@@ -81,13 +81,20 @@ class Enchant
 
     @equipment_manager.empty_hands
     get_item(@brazier) if @use_own_brazier
-    clean_brazier
-    bput("get my #{@item}", 'You get a')
+    
+    case bput("get my #{@item}", 'You get a','That is far too dangerous')
```
Comment (jandersson) [inline enchant.lic]: Does this cover `an|a` item situations?

**Comment by jandersson** [comment]
I think this looks good

File: enchant.lic (hunk 2 above)
Comment (jandersson) [reply enchant.lic]: Noticed there was no space after the `a` so  i will answer my own question with yes :D

**Pull request merged**